python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -137,8 +137,8 @@ class DbtProject(IHaveNew):
 
     Using this helps achieve a setup where the dbt manifest file
     and dbt dependencies are available and up-to-date:
-        * during development, pull the dependencies and reload the manifest at run time to pick up any changes.
-        * when deployed, expect a manifest that was created at build time to reduce start-up time.
```

Comment:
this indentation from https://github.com/dagster-io/dagster/pull/22953  breaks the docs build